import-export-cli/integration/testutils/devFirst_testUtils.go

```diff
@@ -111,6 +113,7 @@ func ValidateInitializeProjectWithDefinitionFlag(t *testing.T, args *InitTestArg
 	//Remove created project
 	t.Cleanup(func() {
 		base.RemoveDir(args.InitFlag)
+		base.WaitForIndexing()
```

Comment:
No need to wait here because we are only removing a directory and nothing related to indexing is taking place in APIM side.